assets/scss/modules/typography/_lists.scss

```diff
@@ -1,17 +1,17 @@
 ul, ol {
   margin: $leading 0 $leading*2 0;
-  padding-left: 1em;
-  list-style-type: square;
+  padding-left: 0px;
+  list-style-type: none;
   letter-spacing: 0.01em;
   list-style-position: inside;
```

Comment:
not sure why I did that originally. I've changed it back to outside. Likewise, the list-style-type was set to none....no idea why that was in there. Might have been something I was thinking about for lists within the callout modules